### PR TITLE
Enhancement/138 update die to who supports saf release

### LINKED ISSUE #138: Update DIE baustatik in who supports SAF
New company finished the implementation - move the company to the `implementation` from `implementation in progress`.

### PR DESCRIPTION
**Pull request by msalak-scia**
description in the issue,
closes #138 as well as #139

### CODE REVIEW

File: docs/getting-started/who-supports-saf.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ See below overview tables which will give you an idea of who supports SAF and to
...
+| RelConnectsRigidCross       |                                  Yes                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                      Yes                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                           |                                          No                                          | Yes | No | No | No |
```
Comment (msalak-scia) [inline docs/getting-started/who-supports-saf.md:47]: These changes are not described in the issue #138, I've noticed there are obsolete information fro SCIA, so fixed them here.

File: docs/getting-started/who-supports-saf.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ See below overview tables which will give you an idea of who supports SAF and to
...
+| RelConnectsRigidMember      |                                  Yes                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                       No                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                          |                                          No                                          | Yes | No | No | No |
```
Comment (msalak-scia) [inline docs/getting-started/who-supports-saf.md:49]: These changes are not described in the issue https://github.com/StructuralAnalysisFormat/StructuralAnalysisFormat-Doc/issues/138, I've noticed there are obsolete information fro SCIA, so fixed them here.

File: docs/getting-started/who-supports-saf.md [hunk 3]
```diff
@@ -67,6 +67,14 @@ See below overview tables which will give you an idea of who supports SAF and to
...
 | StructuralSurfaceActionDistribution |                                  Yes                                 |                                   No                                  |                                            No                                           |                                   No                                  |                                       No                                      |                                           No                                           |                                    No                                   |                                           No                                           |                                          No                                          | No | Yes | No | No |
 
+### Results
```
Comment (msalak-scia) [inline docs/getting-started/who-supports-saf.md:70]: I've added also the issue #139 here - new groups `Results` added

**Review by sonkaste17 — APPROVED** [review]
thx

**Review by sonkaste17 — APPROVED** [review]

**Pull request merged**